### PR TITLE
Clarify when completed status returned by a call

### PR DESCRIPTION
**Pull request by marklewin**
## Description

This change is in response to the following feedback received by @fauna5:

> "completed" is always returned unless the call was rejected. "unanswered", "timeout", "failed", "busy" and "cancelled" all proceed to "completed". The description of the "completed" state probably needs to change to something like "When a call flow has been terminated. This is always received unless the status is 'rejected'"

### CODE REVIEW

File: _documentation/voice/voice-api/guides/call-flow.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Each call goes through a sequence of states in its lifecycle:
 1. Created
```
Comment (fauna5) [inline _documentation/voice/voice-api/guides/call-flow.md]:
```suggestion
1. Started
```

Comment (marklewin) [reply _documentation/voice/voice-api/guides/call-flow.md]: I've reformatted the sequence of events to match the actual event names. I've also added a note saying that the events usually arrive in order but that there are no guarantees and their applications should handle out-of-order messages.

**Comment by marklewin** [comment]
Marking as don't merge while @fauna5 is looking at the timestamps of some out-of-order events he has experienced.

**Comment by fauna5** [comment]
Unfortunately not :( 

CON-0caa8d24-254f-4a33-9bc4-1f4ddb97b220
   status           payload timestamp
4. completed 	2019-04-16T16:24:30.576Z
3. ringing    	2019-04-16T16:24:25.753Z
2. started    	2019-04-16T16:24:25.753Z
1. busy   		2019-04-16T16:24:30.613Z

**Comment by marklewin** [comment]
OK, so are we saying that there is no way to order the events? In which case I'd better remove the suggestion that their application handles it, @mheap?

**Comment by fauna5** [comment]
Best to leave it out i think

**Comment by mheap** [comment]
Which timestamp is out of order? `busy`/`completed`?

**Comment by fauna5** [comment]
Yep

**Comment by mheap** [comment]
I've checked with @JamieChapman8462 and neither order or generation nor order of delivery are guaranteed.

Let's take Rich's suggestion and omit the ordering from the docs

**Pull request merged**